qbcore.sql

```diff
@@ -42,3 +42,13 @@ CREATE TABLE IF NOT EXISTS `player_contacts` (
   PRIMARY KEY (`id`),
   KEY `citizenid` (`citizenid`)
 ) ENGINE=InnoDB AUTO_INCREMENT=1;
+
+CREATE TABLE IF NOT EXISTS `permissions` (
+	`id` INT(11) NOT NULL AUTO_INCREMENT,
+	`name` VARCHAR(255) NOT NULL COLLATE 'utf8mb4_general_ci',
+	`license` VARCHAR(255) NOT NULL COLLATE 'utf8mb4_general_ci',
+	`permission` VARCHAR(255) NOT NULL COLLATE 'utf8mb4_general_ci',
```

Comment:
I think an index between license and permission can be add to prevent multiple permission for the same license no ?